Smalltalk/Vector.som

```diff
@@ -100,7 +100,9 @@ Vector = (
     )
 
     contains: anObject = (
-        ^ storage contains: anObject
+        first to: last - 1 do: [ :i |
+            (storage at: i) = anObject ifTrue: [ ^ true ] ].
```

Comment:
How about:

```suggestion
        self do: [ :element |
            element = anObject ifTrue: [ ^ true ] ].
```

This way, we don't have to worry about the loop being inconsistent.

Reply:
I definitely agree !